browser/plugins/plugins.json

```diff
@@ -146,18 +145,25 @@
         "Object Gaze Clicked": "on_three_object_gaze_clicked"
     },
 
-
     "STATE AND STRUCTURE": {
         "Annotation": "annotation",
+
+        "Is In VR": "is_in_vr",
```

Comment:
'Is in VR' but also our capitalisation overall isn't great ('render To texture' above) so maybe leave for another pass.